code/modules/mob/living/simple_animal/hostile/illusion.dm

```diff
@@ -42,6 +48,17 @@
 		return ..()
 
 
+/mob/living/simple_animal/hostile/illusion/AttackingTarget()
+	..()
+	if(istype(target, /mob/living) && prob(multiply_chance))
+		var/mob/living/L = target
+		if(L.stat == DEAD)
+			return
+		var/mob/living/simple_animal/hostile/illusion/M = new(loc)
+		M.faction = faction
```

Comment:
How much slower is Copy()/is it needed in this particular line? They do all die after 10 seconds after all.

Plus the illusions shouldn't be attacking eachother